### ISSUE TITLE
Bug: application role on container makes escaping from explorer difficult to impossible via screen reader

### ISSUE THREAD

**Issue by RichCaloggero** [mathjax/MathJax#3399]
### Issue Summary
                
When explorer is active, you can tab to each expression and screen reader speaks the label on the container. However, once an expression container whose role is "application" gains focus, all screen reader keys are turned off and you cannot move away from the expression via arrow keys. All you can do is tab, which makes it difficult to continue reading the page after exploring an expression.

### Steps to Reproduce
                
                I'm using NVDA 24.4.2 with either Firefox or Chrome.
                
                1. with screen reader running, goto https://richcaloggero.github.io/test/mathJaxTest.html
                2. from menu, turn on the explorer,  include in tab order, and turn off hidden MathML
                3. tab to one of the expressions
                4. verify you can explor it with the keyboard
                5. press escape, and try and move through the page with arrows 
                
                This is a bug because you cannot escape from the expression's container after it has been focused.
                
                
                ### Technical details
                
                * MathJax Version: 3.2.2
                * Client OS: windows 10
                * Browser: either latest Firefox or Chrome
                
                ## Suggested fix
                
- use `role="tree" aria-roledescription="math"` on the mjx-container elements
   + this role turns off screen reader keys when focused like application role
   + escape turns browse mode back on and allows you to continue arrowing
   + the expression is now announced as "math" rather than "application" which makes more sense

You can see this behavior by clicking the "substitute role" button in the demo cited above.

**Comment by dpvc** [comment]
We released v4 on Monday this week, which has a completely rewritten explorer.  With this explorer, my tests indicate that escape does end focus mode and allow you to continue in browser mode, so you may wish to try that out.

The new explorer does continue to use the application role (though it does announce it as "math" rather than "application", as you suggest, and the description can be changed via menu item).  The tree role was the only other possible choice, but it did not work well across all browser/OS/screen-reader combinations that we need to support, and semantically it is the wrong choice, as that is for making selections among items in a tree structure, and that is not what the explorer is doing.

Getting a solution that works not just for the exploration of the expression but also when reading the page as a whole in the 13 browser/os/screen-reader combinations that we support is a non-trivial task, and one that limits the choices that we have to make.  Some solutions that work beautifully in one setting don't work at all in others.   I agree that the v3 experience was not ideal, but I think the new v4 explorer setup should give a smoother experience across the setups that we target.
